internal/kafka/internal/presenters/cluster.go

```diff
@@ -18,11 +18,22 @@ func PresentEnterpriseCluster(cluster api.Cluster, fleetShardParams services.Par
 		})
 	}
 
-	c := public.EnterpriseCluster{
+	c := public.EnterpriseClusterRegistrationResponse{
```

Comment:
I am wondering if we should do something similar like https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/af1690e0ad9498041edca7dc08c3843f41b2690d/internal/kafka/internal/presenters/cluster.go#L31 over here as well?

Reply:
yes. will add it as well